candybot/candybot/commands/admin/blacklist.py

```diff
@@ -1,21 +1,27 @@
-from candybot.interface import database
+from candybot.interface import database, converters
 from candybot.commands.framework import AdminCommand, ArgumentSpec, UserArgument
 
 
 class BlacklistCommand(AdminCommand):
     name = "blacklist"
-    help = "Blacklists a user from interacting with CandyBot."
+    help = "Shows current blacklist or blacklists a user from interacting with CandyBot."
     aliases = ["bl"]
-    examples = ["@User", "User#1234", "123456789"]
-    argument_spec = ArgumentSpec([UserArgument], False)
+    examples = ["", "@User", "User#1234", "123456789"]
+    argument_spec = ArgumentSpec([UserArgument], True)
     clean = True
     ignore = False
-
+    
     async def _run(self):
-        blacklist = database.get_blacklist(self.message.guild.id)
-        if self.user.id in blacklist:
-            database.set_blacklist(self.message.guild.id, self.user.id, remove=True)
-            await self.send(f"{self.user.mention} was removed from the CandyBot blacklist")
+        blacklist = database.get_blacklist()
+        if self.user is None:
+            self.title = ":lock: CandyBot Blacklist"
+            # How to get user mentions from DB?
+            bl_users = [converters.to_user(x, self.message.guild).mention for x in blacklist]
```

Comment:
This needs to be `[(await converters.to_user(x, self.message.guild)).mention for x in blacklist]` because ` converters.to_user doesnt return a value, but a co-routine.